### ISSUE TITLE
Maintainers wanted

### ISSUE THREAD

**Issue by bbaassssiiee** [ansible-collections/ansible-consul#526]
This repository needs more TLC than I have time for. Are there any volunteers to take the baton?

**Comment by nre-ableton** [comment]
@bbaassssiiee Yes, I would be happy to assist. My company already maintains [several OSS Ansible roles](https://galaxy.ansible.com/ableton). We would really like to see some cleanups and improvements made in this role as well.

**Comment by nre-ableton** [comment]
@bbaassssiiee Thanks!

**Comment by nre-ableton** [comment]
@bbaassssiiee Unfortunately, I'm reopening this issue and asking to be removed from the list of maintainers. I don't have much time to work on the role these days, and we're no longer using consul in our infrastructure anymore.

**Comment by bbaassssiiee** [comment]
Too bad! HashiCorp doesn't want to provide resources either. And I don't use Consul anymore.

Any other volunteers? Otherwise we can archive this repository.

**Comment by bbaassssiiee** [comment]
> @bbaassssiiee Unfortunately, I'm reopening this issue and asking to be removed from the list of maintainers. I don't have much time to work on the role these days, and we're no longer using consul in our infrastructure anymore.

Nik, thanks for all your help! I removed you from the maintainers.

**Comment by rismalrv** [comment]
> Too bad! HashiCorp doesn't want to provide resources either. And I don't use Consul anymore.
> 
> Any other volunteers? Otherwise we can archive this repository.

i'm not sure this is the right place to ask. care to share why not using consul anymore? found better approach? lol

**Comment by bbaassssiiee** [comment]
Found other job

**Comment by ahjohannessen** [comment]
Who are the maintainers now? Only @nre-ableton ?

**Comment by nre-ableton** [comment]
> Who are the maintainers now? Only [@nre-ableton](https://github.com/nre-ableton) ?

Sorry, no. Please see [my above post](https://github.com/ansible-collections/ansible-consul/issues/526#issuecomment-2124272391). My company no longer uses consul, and I no longer have administrative access to this repository.

**Comment by bbaassssiiee** [comment]
Thanks for all your help Nik!
